internal/services/controller/handlers/filters/autoscalerevents/filter.go

```diff
@@ -0,0 +1,18 @@
+package autoscalerevents
+
+import (
+	"castai-agent/internal/castai"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	AutoScalerEvent = "autoscaler.cast.ai"
```

Comment:
Lets rename it to `AutoscalerEvent` to keep it consistent. We treat "autoscaling" as a single word.

Reply:
Also, it might be better sematically to call it `AutoscalerController`